src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -54,14 +54,29 @@
                               Margin="0,2,0,2"
                               Style="{StaticResource SchemeGridStyle}">
                             <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto" />
                                 <ColumnDefinition Width="*" />
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
                             <TextBlock Grid.Column="0"
                                        VerticalAlignment="Center"
                                        Style="{StaticResource SettingsPageItemHeaderStyle}"
                                        Text="{x:Bind Name, Mode=OneWay}" />
-                            <Grid Grid.Column="1"
+                            <Border Grid.Column="1"
+                                    Margin="10,0,0,0"
+                                    Padding="2,0,2,0"
+                                    HorizontalAlignment="Left"
+                                    VerticalAlignment="Center"
+                                    Background="{ThemeResource SystemAltMediumLowColor}"
+                                    BorderBrush="{ThemeResource SystemControlForegroundBaseMediumBrush}"
+                                    BorderThickness="1"
+                                    CornerRadius="1"
+                                    Visibility="{x:Bind IsDefaultScheme, Mode=OneWay}">
+                                <TextBlock x:Uid="ColorScheme_DefaultTag"

```

Comment:
should we give this a better Narrator description? "This is the current default color scheme"

Reply:
This is the only one I couldn't figure out. The "default" text is "focusable" by narrator so it should be somewhat clear which item is the default.

Ideally, the help text would be set like you said, but I'm having trouble accessing the `ListViewItem` and setting the help text on it. This help text would then have to be updated each time the default tag moves. I think this makes sense as a follow-up since the solution is going to be fairly ugly.

Reply:
moved to #13891